aria/modeling/exceptions.py

```diff
@@ -22,9 +22,9 @@ class ModelingException(AriaException):
     """
 
 
-class InputsException(ModelingException):
+class ParameterException(ModelingException):
```

Comment:
I'm not sure I understand this change - It's true that the model object is `Parameter` and not `Inputs` (although that too is supposedly under change by @AviaE ) - but the error itself is not about the model type and doesn't have anything much to do with the models themselves - it's about inputs, so why shouldn't the exception be named like so?